beetsplug/convert.py

```diff
@@ -100,8 +100,18 @@ def should_transcode(item, fmt):
             query, _ = parse_query_string(query_string, Item)
             if query.match(item):
                 return False
-    if config['convert']['never_convert_lossy_files'] and \
-            not (item.format.lower() in LOSSLESS_FORMATS):
+
+    try:
```

Comment:
This is a try / except block, which apparently cause some controversy when used in Python. Elsewhere in `convert.py`, it [enumerates keys](https://github.com/beetbox/beets/blob/5f0d7c6d0a3f95846ad1d28f766aecdb34775580/beetsplug/convert.py#L79) and checks if certain keys exist. I couldn't get that working, but could try again if needed.

Reply:
Indeed, it's generally a good idea to avoid what's called a "bare except," where the `except:` line doesn't specify the type of exception that's expected. The problem is that it can silence exceptions you really want to know about, like a `NameError` if you misspell the name of a variable.

Using a try/except block is fine, but if you know what exception you're looking for, you can specify that—for example, `except KeyError:`.